testsuite/python/rotation_per_particle.py

```diff
@@ -28,47 +28,49 @@ class Rotation(ut.TestCase):
     s.cell_system.skin = 0
     s.time_step = 0.01
 
+    def tearDown(self):
+        self.s.part.clear()
+
     def test_langevin(self):
         """Applies langevin thermostat and checks that correct axes get
            thermalized"""
         s = self.s
         s.thermostat.set_langevin(gamma=1, kT=1, seed=42)
-        for x in (0, 1):
-            for y in (0, 1):
-                for z in (0, 1):
+        for x in (False, True):
+            for y in (False, True):
+                for z in (False, True):
```

Comment:
I would suggest to replace the three loops with 
`for x, y, z in itertools.product((False, True), repeat=3):`

Reply:
good idea!